public/style/sass/_browse.scss

```diff
@@ -9,7 +9,7 @@
 
 // style leaflet geosearch search box
 .my-map form {
-  width: calc(100vw - 55px);
+  width: calc(65vw - 55px);
```

Comment:
set this in report.scss instead and change it back to 100vw here, since it messes up the search bar on the browse page - could maybe change it to `.my-map.browse form` as well to make it specific to the browse view